src/NServiceBus.Autofac/NServiceBus.Autofac.csproj

```diff
@@ -18,6 +18,7 @@
     <PackageReference Include="NServiceBus" Version="[7.0.0, 8.0.0)" />
     <PackageReference Include="Fody" Version="6.1.0" PrivateAssets="All" />
     <PackageReference Include="Janitor.Fody" Version="1.8.0" PrivateAssets="All" />
+    <PackageReference Include="Obsolete.Fody" Version="5.2.0" />
```

Comment:
Missing PrivateAssets="All"

Reply:
Good catch, I wonder if we should have an approval file for new deps on the production package to break the build for mistakes like this?

Reply:
Not entirely sure how that would work since you'd have to dig into the obj folder to find the generated nuspec file.

Reply:
couldn't we save all the packagereferences from the csproj which don't have the privateassets="all" to a text file and compare those?

Reply:
Yea, sounds that it could work? Perhaps something for `Particular.Approvals`?